### ISSUE TITLE
PR creation for new entities fails validation through the API

### ISSUE THREAD

**Issue by caufieldjh** [bridge2ai/b2ai-standards-registry#25]
Issues created with the `New` tag are successfully processed: their proposed changed are committed to a newly created branch and a new branch is created for the PR (e.g., `create-pull-request/patch-5797756`).
A new PR is not created for this branch. Why not?
This is what the action log says:
```
  Attempting creation of pull request
  Error: Validation Failed: {"resource":"Issue","code":"missing_field","field":"title"}
```
This should be handled by the `make_title` function.

**Comment by caufieldjh** [comment]
Still failing - see https://github.com/bridge2ai/b2ai-standards-registry/actions/runs/4387795879

**Comment by caufieldjh** [comment]
Wondering if this is an environment issue related to poetry: the script sets these vars -
```
    if github:
        click.echo(
            f"""
          ::set-output name=PR_BODY::{body}
          ::set-output name=PR_TITLE::{title}
        """
        )
```

But that script runs under a poetry environment:
https://github.com/bridge2ai/b2ai-standards-registry/blob/656ed11840f27c3ff0e41902c1fb1472036e2fea/.github/workflows/new_entity_pr.yml#L27

So when the rest of the action runs, it tries to access those body and title values but they aren't set.
How else may they be passed?

**Comment by caufieldjh** [comment]
Wasn't a poetry issue, but it *was* an environment vars issue - the workflow YAML was referencing the vars with the incorrect IDs.
Fixed in #34
